UsersController.cs

```diff
@@ -0,0 +1,9 @@
+[HttpPost("delete/{id}")]
+public void Delete(uint id)
+{
+    User user = _context.Users.FirstOrDefault(user => user.Id == id);
+    _context.Users.Remove(user);
+    _context.SaveChanges();
+    Debug.WriteLine($"The user with Login={user.login} has been deleted.");
+    return Ok();
```

Comment:
Preferable return type for successful delete action would be a `NoContent()`.
```suggestion
    return NoContent();
```